alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/ODFERestTestCase.kt

```diff
@@ -72,20 +79,48 @@ abstract class ODFERestTestCase : ESRestTestCase() {
                 val indexName: String = jsonObject["index"] as String
                 // .opendistro_security isn't allowed to delete from cluster
                 if (".opendistro_security" != indexName) {
-                    client().performRequest(Request("DELETE", "/$indexName"))
+                    adminClient().performRequest(Request("DELETE", "/$indexName"))
                 }
             } }
     }
+
+    /**
+     * Returns the REST client settings used for super-admin actions like cleaning up after the test has completed.
+     */
+    override fun restAdminSettings(): Settings {
+        return Settings
+                .builder()
+                .put("http.port", 9200)
+                .put(OPENDISTRO_SECURITY_SSL_HTTP_ENABLED, true)
+                .put(OPENDISTRO_SECURITY_SSL_HTTP_PEMCERT_FILEPATH, "sample.pem")
+                .put(OPENDISTRO_SECURITY_SSL_HTTP_KEYSTORE_FILEPATH, "test-kirk.jks")
+                .put(OPENDISTRO_SECURITY_SSL_HTTP_KEYSTORE_PASSWORD, "changeit")
+                .put(OPENDISTRO_SECURITY_SSL_HTTP_KEYSTORE_KEYPASSWORD, "changeit")
```

Comment:
Out of curiosity, what's the difference between the `KEYSTORE_PASSWORD` and `KEYSTORE_KEYPASSWORD`?